renderer/viewer/three/waypointSprite.ts

```
@@ -343,7 +353,7 @@ export function createWaypointSprite (options: {
 }
 
 // Internal helpers
-function drawCombinedCanvas (color: number, id: string, distance: string): HTMLCanvasElement {
+function drawCombinedCanvas (color: number, id: string, distance: string, visualScale: number = 1.0): HTMLCanvasElement {
```

ESLint

[error] Type number trivially inferred from a number literal, remove type annotation.

(`@typescript-eslint/no-inferrable-types`)

---

[error] Don't use a zero fraction in the number.

(unicorn/no-zero-fractions)